entries/p/polisystems.ch.json

```
@@ -0,0 +1,14 @@
+{
+   "Poli Systems":{
+      "domain": "polisystems.ch",
+      "tfa": [
+         "totp"
+      ],
+      "documentation": "https://wiki.polisystems.ch/English/Tutorials/Two-factor-authentification-on-Poli-Systems-Portal/",
+      "keywords": [
+         "hosting",
+         "cloud",
+         "email"
```

> High-end mail hosting _included_ in your web hosting, ...

As long as they don't offer standalone mail hosting, we should not include `email` here, I think.
```suggestion
         "cloud"
```